apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-08-03T21:27:05Z"
+  build_date: "2022-08-11T19:40:37Z"
   build_hash: 87477ae8ca8ac6ddb8c565bbd910cc7e30f55ed0
   go_version: go1.18.3
   version: v0.19.3-dirty
-api_directory_checksum: 57b3efac72bde4af8b7f35a3d6769314bb1e7d7c
+api_directory_checksum: 2ed2eed46f763f970340c40e89f51f9d994c03c5
```

Comment:
Weird, why did the `api_directory_checksum` change? i guess maybe we forgot a regeneration from a previous PR